lib/dal/src/workspace_snapshot/graph.rs

```diff
@@ -899,7 +899,9 @@ impl WorkspaceSnapshotGraph {
         // ```
         // pbpaste | dot -Tsvg -o foo.svg && open foo.svg
         // ```
-        let current_root_weight = self.get_node_weight(self.root_index).unwrap();
+        let current_root_weight = self
+            .get_node_weight(self.root_index)
+            .expect("this should be impossible and this code should only be used for debugging");
```

Comment:
We allow expects in debug output functions. This is within a function to prints out a "dot" graph for local development only.